webpack.config.js

```diff
@@ -1,6 +1,5 @@
 const path = require('path');
-const webpack = require('webpack');
-const pkg = require('./package.json');
+const TerserPlugin = require('terser-webpack-plugin');
 
```

Comment:
With upgrading to webpack v4, we need to add the `mode` key to this configuration https://webpack.js.org/concepts/mode/